docs/src/booklet/2-qubo.md

```diff
@@ -31,14 +32,10 @@ Moreover, [Ising Model](https://en.wikipedia.org/wiki/Ising_model).
 \end{array}
 ```
 
-With that said, the significant advances in computing systems and algorithms specialized for sampling QUBOs have contributed to their popularity.
-
-Some of the paradigms that stand out for running QUBOs are quantum gate-based optimization algorithms (QAOA and VQE), quantum annealers and hardware-accelerated platforms (Coherent Ising Machines and Simulated Bifurcation Machines).
+where ``J \in \mathbb{R}^{n \times n}`` is strictly upper triangular and ``\mathbf{h} \in \mathbb{R}``.
 
 [![D-Wave Washington 1000Q](../assets/figures/quantum-chip.png)](https://commons.wikimedia.org/wiki/File:D-Wave-Washington-1000Q.jpg)
 
-```@docs
-ToQUBO.isqubo
-ToQUBO.toqubo
-ToQUBO.toqubo!
-```
+The Ising reformulation alternative draws the bridge between QUBO problems and specialized devices that rely on the laws of physics to compute solution candidates.
```

Comment:
That *may* rely on laws of nature. Let's use the Ising solvers definition from the Ising solver review (McMahon and group) and add a citation to them